.github/workflows/ci-tests.yml

```diff
@@ -60,7 +62,38 @@ jobs:
         distribution: 'zulu'
         java-version: ${{ matrix.java-version }}
     
-    - name: Checkout Freenet git repository
+    - name: Load and print current fred branch next HEAD commit
+      shell: bash
+      run: |
+        set -o errexit
+        set -o pipefail
+        echo 'Current HEAD commit of fred branch next:'
+        git ls-remote 'https://github.com/freenet/fred.git' 'refs/heads/next' | cut -f 1 \
+          | tee current_fred_commit
+    
+    - name: Use/populate cache of fred JARs
+      id: cache
+      uses: actions/cache@v3
+      with:
+        path: |
+          fred/build/output/
+          fred/gradle/  # fred's Gradle binaries are also used for Freetalk's compilation
```

Comment:
is this treated as a comment?

Reply:
I think if it wasn't treated as a comment then the code wouldn't have fixed the bug which I wrote it for to fix in commit 849d2ee94474bf2ec33a29041289b8f4642fe45d

But the code has been removed by PR #56 now for unrelated reasons so it doesn't matter anymore anyways.